issues/8511/112 Bücher.html

```diff
@@ -86,59 +86,15 @@ <h2>Mathematische Routinen VC 20, Elektrotechnik/Elektronik</h2>
 
-        <p>Je weiter man in der Lektüre dieses Buches kommt, desto erfreulicher wird es allerdings. Die Programme zur Zahlenkon vertierung, zur komplexen Arithmetik und Trigonometrie sind sauber geschrieben und überschaubar dokumentiert. Zwei weitere Routinen zum Ermitteln der Nullstellen einer Funktion und zur Berechnung bestimmter Integrale nach der Trapezregel verdienen lediglich den Tadel, daß die Funktionseingabe platzraubend und schwerfällig ist. Die Möglichkeit, über den Tastaturpuffer zu arbeiten, wäre wesentlich eleganter gewesen. Der mathematische Teil wird abgerundet durch ein Programm zur Lösung linearer Gleichungssysteme mittels des Gaußschen Algorithmus und ein letztes zur Polynomapproximation nach der Methode der kleinsten Fehlerquadrate.</p>
+        <p>Je weiter man in der Lektüre dieses Buches kommt, desto erfreulicher wird es allerdings. Die Programme zur Zahlenkonvertierung, zur komplexen Arithmetik und Trigonometrie sind sauber geschrieben und überschaubar dokumentiert. Zwei weitere Routinen zum Ermitteln der Nullstellen einer Funktion und zur Berechnung bestimmter Integrale nach der Trapezregel verdienen lediglich den Tadel, daß die Funktionseingabe platzraubend und schwerfällig ist. Die Möglichkeit, über den Tastaturpuffer zu arbeiten, wäre wesentlich eleganter gewesen. Der mathematische Teil wird abgerundet durch ein Programm zur Lösung linearer Gleichungssysteme mittels des Gaußschen Algorithmus und ein letztes zur Polynomapproximation nach der Methode der kleinsten Fehlerquadrate.</p>
 
         <p>Im dritten Teil des Buches findet der Elektroniker vieles, was sein Herz begehrt. Man merkt, daß der Autor hier in seinem Fachgebiet programmiert. 14 Programme aus dem Bereich der Elektrotechnik und Elektronik werden vorgestellt, von denen hier nur erwähnt seien: Berechnung einlagiger Zylinderspulen (es wäre benutzerfreundlich gewesen, die Änderung der relativen Permeabilität einfacher zu gestalten), Schwingkreis- und Reaktanzberechnungen, mehrere Routinen zur Berechnung von passiven und aktiven Tief-, Hoch- und Bandpässen, Berechnungen zur HF-Leitungstechnik und als Höhepunkt eine Netzwerkanalyse.</p>
 
         <p>Fazit: Das Buch ist eine Fundgrube für Studenten oder Ingenieure der Elektrotechnik und Elektronik. ZTT .</p>
 
         <address class="author">(Heimo Ponnath)</address>
 
-        <p class="source">Info: Ernst-Friedrich Reinking, Mathematische Routinen VC 20, Elektrotechnik/Elek-tronik, Vieweg &amp; Sohn 1983, ISBN 3-528-04234-6, 29,90 Mark</p>
-
-        <h2>C 64 Computer-Handbuch</h2>
```

Comment:
In addition to "Listing des Monats" and "Anwendung des Monats", which have recently been in two articles that we combined, this issue has two more articles that came in two parts: "Bücher" and "Apfelmännchen". Since I consider this a freedom taken by the typesetter, I combined them.

You're removing the second part of Bücher hier – where does it go?

Reply:
Ok, okay, if both the TOC and the Index agree, let's seep it separate!